### ISSUE TITLE
Crashes on startup due to `ci_activity`

### ISSUE THREAD

**Issue by casperdcl** [slapperwan/gh4a#975]
```
# logcat -b crash
FATAL EXCEPTION: main
Process: com.gh4a, PID: 20519
io.reactivex.exceptions.CompositeException: 1 exceptions occurred.
       at io.reactivex.internal.observers.ConsumerSingleObserver.onError(ConsumerSingleObserver.java:47)
       at com.philosophicalhacker.lib.ReactiveLoaders$LoaderSingle$1.onError(ReactiveLoaders.java:51)
       at com.philosophicalhacker.lib.RxLoaderCallbacks.onLoadFinished(RxLoaderCallbacks.java:33)
       at android.support.v4.app.LoaderManagerImpl$LoaderInfo.callOnLoadFinished(LoaderManager.java:476)
       at android.support.v4.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManager.java:444)
       at android.support.v4.content.Loader.deliverResult(Loader.java:126)
       at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.safeDeliverResult(RxLoaderCallbacks.java:109)
       at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.access$000(RxLoaderCallbacks.java:44)
       at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader$2.accept(RxLoaderCallbacks.java:85)
       at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader$2.accept(RxLoaderCallbacks.java:82)
       at io.reactivex.internal.observers.ConsumerSingleObserver.onError(ConsumerSingleObserver.java:44)
       at io.reactivex.internal.operators.single.SingleObserveOn$ObserveOnSingleObserver.run(SingleObserveOn.java:79)
       at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7063)
       at java.lang.reflect.Method.invoke(Native Method)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
  ComposedException 1 :
       com.squareup.moshi.JsonDataException: Expected one of [subscribed, manual, author, comment, mention, review_requested, team_mention, state_change, security_alert, assign] but was ci_activity at path $.items[0].reason
          at com.squareup.moshi.StandardJsonAdapters$EnumJsonAdapter.fromJson(StandardJsonAdapters.java:244)
          at com.squareup.moshi.StandardJsonAdapters$EnumJsonAdapter.fromJson(StandardJsonAdapters.java:213)
          at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:127)
          at com.meisolsson.githubsdk.model.$AutoValue_NotificationThread$MoshiJsonAdapter.fromJson($AutoValue_NotificationThread.java:75)
          at com.meisolsson.githubsdk.model.$AutoValue_NotificationThread$MoshiJsonAdapter.fromJson($AutoValue_NotificationThread.java:28)
          at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:127)
          at com.squareup.moshi.CollectionJsonAdapter.fromJson(CollectionJsonAdapter.java:75)
          at com.squareup.moshi.CollectionJsonAdapter$2.fromJson(CollectionJsonAdapter.java:52)
          at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:127)
          at com.meisolsson.githubsdk.model.AutoValue_Page$MoshiJsonAdapter.fromJson(AutoValue_Page.java:62)
          at com.meisolsson.githubsdk.model.AutoValue_Page$MoshiJsonAdapter.fromJson(AutoValue_Page.java:20)
          at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:127)
          at com.squareup.moshi.JsonAdapter.fromJson(JsonAdapter.java:34)
          at retrofit2.converter.moshi.MoshiResponseBodyConverter.convert(MoshiResponseBodyConverter.java:42)
          at retrofit2.converter.moshi.MoshiResponseBodyConverter.convert(MoshiResponseBodyConverter.java:25)
          at retrofit2.ServiceMethod.toResponse(ServiceMethod.java:118)          at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:212)
          at retrofit2.OkHttpCall.execute(OkHttpCall.java:174)
          at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:41)
          at io.reactivex.Observable.subscribe(Observable.java:10700)
          at io.reactivex.internal.operators.observable.ObservableSingleSingle.subscribeActual(ObservableSingleSingle.java:35)
          at io.reactivex.Single.subscribe(Single.java:2692)
          at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:33)
          at io.reactivex.Single.subscribe(Single.java:2692)
          at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:33)
          at io.reactivex.Single.subscribe(Single.java:2692)
          at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
          at io.reactivex.Scheduler$1.run(Scheduler.java:138)
          at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:59)
          at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:51)
          at java.util.concurrent.FutureTask.run(FutureTask.java:266)
          at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
          at java.lang.Thread.run(Thread.java:764)
```

**Comment by casperdcl** [comment]
Might be duplicate of #956? Seems to have resurfaced...

**Comment by casperdcl** [comment]
ah nvm; looks like it is #956, and will be pushed/fixed by #970
